IowaPD_people.yaml

```diff
@@ -572,9 +646,15 @@ entityDefinitions:
       contact.phonenumber:
         type: 'contact.phonenumber'
         column: 'CellPhone'
-  JohnsonCounty_IowaCity_Dispatch:
-    fqn: 'ol.dispatch'
-    entitySetName: 'JohnsonCounty_IowaCity_Dispatch'
+      contact.cellphone:
+        type: contact.cellphone
+        transforms:
+        - !<transforms.ValueTransform>
+          value: True
```

Comment:
Do we need a condition here to write in only when cellphone is not null? Because of this value transform.